doc/aptinstall.sh

```diff
@@ -18,6 +18,8 @@ sudo apt-get install\
   texlive-fonts-recommended\
   texlive-fonts-extra\
   texlive-xetex\
+  xindy\
+  xindy-rules\
```

Comment:
I hesitated whether I should update `doc/src/contributing/new-contributors-guide/build-docs.rst` as well, because it does not seem complete already (no mention of `texlive-xetex` there).

An alternative is to use `latex_use_xindy = False` in conf.py. This will keep using `makeindex` so the Index will look same as so far.  Seemingly there aren't Unicode characters in there.  Hmm maybe it is better I push this rather.  Only waiting to see if test-sphinx succeeds with xindy, then I will use `latex_use_xindy = False`.

Reply:
The [test-sphinx](https://github.com/sympy/sympy/actions/runs/10170020818/job/28128916368?pr=26868) succeeded with xindy.

For reasons of not adding dependencies needing documentation I will push `latex_use_xindy = False` rather.

Reply:
You can also update the build docs page. The pdf build instructions aren't maintained as well because not very many people build the pdf docs, but there's no reason they shouldn't be accurate.

Reply:
After pondering the matter I simpliy pushed `latex_use_xindy = False` thus sparing me the additional updates to build docs and dependencies. I would never have been able anyhow to document what is needed on Windows.

Reply:
I didn't even realize the pdf docs could be built on Windows. It isn't a high priority if we need some package to make them better. Most people don't need to build the pdf docs locally. It's mainly just important that they can be built on the CI so that they get compiled when we do a release (plus the PDF docs have a the nice side effect of catching invalid LaTeX math examples).